src/test/resources/features/booking_search.feature

```diff
@@ -13,8 +13,8 @@ Feature: Booking Search
 
   Scenario: Search all offenders
     When a booking search is made without any criteria
-    Then "10" booking records are returned
```

Comment:
If we do care about these numbers then using shared test data is not the way to achive this, so I'm assuming not.